tests/pythonMeep.bash

```diff
@@ -0,0 +1,8 @@
+#/bin/bash
+
+#
+# test for successful loading of python meep module
+#
+cd ../python
+ln -f -s .libs/_meep.so
+python -c 'import meep'
```

Comment:
This should go directly into the Makefile.am.  The Makefile is already running the shell, so it's a bit perverse to have a separate file for this.